### ISSUE TITLE
Could'nt find function lra in symbol table 

### ISSUE THREAD

**Issue by justwagle** [STEllAR-GROUP/phylanx#430]
Trying to use `lra_csv_instrumented` with `--direct` option results in the following error.
```
{what}: lra(9, 9): couldn't find function 'lra' in symbol table: HPX(bad_parameter)

terminate called after throwing an instance of 'hpx::detail::exception_with_info<hpx::exception>'
  what():  lra(9, 9): couldn't find function 'lra' in symbol table: HPX(bad_parameter)
Aborted
```

**Comment by khuck** [comment]
@justwagle any solution to this?  I am getting a similar error when I build/run lra_csv on OSX: 
```
{what}: read_x(1, 1): couldn't find function 'block' in symbol table: HPX(bad_parameter)
```

**Comment by justwagle** [comment]
@khuck The issue I created I was able to fix with #431 . In this case the plugin was simply not being loaded correctly due to bad cmake. However, the issue you mentioned where it is  not finding the 'block' symbol looks like it is related to https://github.com/STEllAR-GROUP/phylanx/issues/402. @ct-clmsn found the following workaround for it 

> i had to export/assign LD_PRELOAD with a colon delimited list of the shared libraries from $INSTALL_DIR/lib/phylanx/ before running the python3 interpreter with the phylanx_lra and phylanx_als scripts. everything works fine now.
